shell/pom.xml

```diff
@@ -69,5 +69,45 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
I don't know it is necessary to introduce ufs dependencies to alluxio shell module. 
Shell module does not generate shaded jar. So putting dependencies here will not help any way.